docs/src/developer/style.md

```diff
@@ -83,52 +119,114 @@ Ternary operators provide a way to write clean, concise code.  Use good
 judgement.
 
 Good:
->
-    y = x > 0 ? x : -x
+
+```julia
+y = x > 0 ? x : -x
+```
 
 There are many examples in our codebase that use the form ```<cond> ?
 <statement> : <statement>```.  These can be expressed much more clearly in an
 if/else statement.
 
 ## Unit Tests
+
 All code should be tested.
 
 ## Whitespace
+
 * If many function arguments cause the line length to be exceeded, put one
 argument per line. In some cases it may make sense to pair some variables on
 the same line.
->
-    function foo(var1::String,
-                 var2::String,
-                 var3::String,
-                 var4::String,
-                 var5::String,
-                 var6::String)
-
-* Do not surround equal signs with spaces when passing keyword args to a
+
+Good:
+
+```julia
```

Comment:
@claytonpbarrows @jd-lara Are you guys OK with this?  I like this a lot better than what we've been doing because we won't have to adjust whitespace whenever the function name changes.

Reply:
I am indifferent to this choice.

Reply:
It seems like we are in agreement.